ui/src/layouts/storage/functions.rs

```diff
@@ -48,8 +48,17 @@ pub fn run_verifications_and_update_storage(
     }
 }
 
-// #[cfg(not(target_os = "macos"))]
+pub fn get_items_from_current_directory(cx: &Scoped<Props>, ch: &Coroutine<ChanCmd>) {
+    use_future(cx, (), |_| {
+        to_owned![ch];
+        async move {
+            sleep(Duration::from_millis(300)).await;
```

Comment:
300ms seems like a very short time. We're not processing hardware interrupts here. it's probably fine if it takes a second or two for uplink to detect a new file.